### PR TITLE
Created "gitignore" for Unreal Engine 4 projects

### PR DESCRIPTION
**Pull request by chbecher**

### CODE REVIEW

File: UE4.gitignore [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+*.ipa
+*.apk
+
+# Generated by the engine
+*.xcodeproj
+*.sln
+*.suo
+*.opensdf
+*.sdf
+*.sln
```
Comment (shiftkey) [inline UE4.gitignore]: Duplicate

**Comment by shiftkey** [comment]
This all seems nice. 

Thoughts @arcresu?

**Comment by arcresu** [comment]
LGTM. Just that I usually prefer to alphabetically sort the rules under each sub-heading which helos with catching duplicates and overlapping rules.

**Comment by shiftkey** [comment]
@chbecher looks like we're good to go after addressing this feedback!

**Comment by chbecher** [comment]
I am on it :)

**Comment by shiftkey** [comment]
It looks like there's a few extensions which have been removed - could you shed some light on the motivation for dropping those?

**Comment by chbecher** [comment]
Sorry, that was a misunderstanding with the duplicates. 
I'll add the "duplicates" again and sort them alphabetically under the heading. Right?
I used as example the Unity.gitignore
Do you have a better example?

**Comment by shiftkey** [comment]
Closing in favour of #1719 which has some additional rules.

@chbecher apologies for not returning to this PR in a timely manner.

**Pull request closed without merging**